docs/try-viam/rover-resources/rover-tutorial-fragments.md

```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
 
 {{<imgproc src="try-viam/rover-resources/fragments/fragment_configuration.png" resize="1200x" alt="Fragment configuration">}}
 
-The fragment adds the following components to your robot's JSON configuration:
+The fragment adds the following components to your robot's JSON configuration.
+For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component below.
+To see the pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
 
-* [Board component](/components/board/), which is the Raspberry Pi.
-  * Within the board component attributes, digital interrupts: "re" to pin "37" and "le" to pin "35" and I2Cs: name "default_i2c_bus" and bus "1".
-* Right gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Renc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "16 GPIO 23", B/In2: "18 GPIO 24", PWM: "22 GPIO 25".
-  * Depends on local and Renc.
-* Left gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Lenc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "11 GPIO 17", B/In2: "13 GPIO 27", PWM: "15 GPIO 22".
-  * Depends on local and Lenc.
+- A [board component](/components/board/pi/) named `local` representing the Raspberry Pi
+  - An I<sup>2</sup>C bus is configured on the board because the accelerometer connects to the board with I<sup>2</sup>C.
+- Two [motors](/components/motor/gpio/) (`right` and `left`)
+  - The configured pin numbers correspond to where the motor drivers are connected to the board.
+- Two [encoders](/components/encoder/single/), one for each motor
+- A wheeled [base](/components/base/), an abstraction that coordinates the motors for steering
```

Comment:
I think adding the following attributes here would be helpful:
* Wheel circumference (mm): 217
  * Width (mm): 260
  * Spin slip factor: 1
  
Yes they are in the configuration but without looking there they're values that aren't immediately obvious without the user doing work

Reply:
the same goes for the motor. Let's add those values on the list.

Reply:
```suggestion
- A wheeled [base](/components/base/), an abstraction that coordinates the movement of the right and left motors.
The width between the wheel centers is 260mm, the wheel circumference is 217mm, and the spin slip factor is 1.
```
How about that?